glossary.md

```diff
@@ -81,17 +81,6 @@ Readme files pertain to the contents of a directory or the scope of a repository
 * Support expectations: Is this an experimental or maintained project.  Include our standard definition and terms for the given state.
 * Limitations: Is ARM supported? What Equinix Metal features or service flags are required? Are these features available in all regions? The following features are not available in all facilities, for example storage, global ipv4, backend transfer.
```

Comment:
Not sure why ARM is specifically mentioned.

Reply:
This text is derived from another document. That document uses this question ("Does it work on ARM?") as an example of a limitation that could be highlighted in a README.md.

Much of that document was restructured into this glossary.  I don't know that the glossary approach fits this project, especially with the wider goals and audience.

As `equinix-labs/equinix-labs` (rather than `packethost/standards`), "Metal" specific call-outs may not be as helpful as they once were.

Reply:
The purpose-and-governance.md file refers to README.md, OWNERS.md. CODEOWNERS, etc. We could link to this glossary, which gives this project the kind of cohesion we were thinking of by combining standards and equinix-labs.